src/Website/Controllers/ContactUsController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using AttributeRouting.Web.Mvc;
+
+namespace mbsoft.BrewClub.Website.Controllers
+{
+    public class ContactUsController : Controller
+    {
+        [GET("ContactUs")]
```

Comment:
My personal preference with URLs is for them to be all lower case, unless we're hitting something like say an URL which contains a username or something.  We should decide on a standard.